src/ExRender.hs

```diff
@@ -112,7 +114,8 @@ instance ExRenderPackage GenericPackageDescription where
                                  "has_hscolour=false",
                                  "has_profile=false"
                                  ]
-                exParams = spaces $ exHasLib <+> exHasBin <+> exHasOptions
+                exHackageRev = maybe empty (\r -> text $ "rev=" ++ r) hackageRev
```

Comment:
Hint: `(\r -> text $ "rev=" ++ r)` can be written as `(text . ("rev=" ++))`. Once I was told that this is called eta-reduction :)

Reply:
I know what eta-reduction is. I find it less readable.